### ISSUE TITLE
Adding few feature to EVENTS and PROGRAMS.

### ISSUE THREAD

**Issue by Nikita110602** [Opentek-Org/opentek#220]
Adding few features in events and programs so that user can know what are the events there are and go for it . By adding features and events we can make it a broad project and usable for user.
Add the link to the EVENT page and PROGRAM page
![Screenshot (39)](https://github.com/Opentek-Org/opentek/assets/111515819/6e88ef0d-247b-4d1f-9c09-05b2fd6e0913)

**Comment by Nikita110602** [comment]
Pls assign this work to me . I want to contribute in open source.

**Comment by sunny0625** [comment]
Okay, the issue is assigned to you!

**Comment by sunny0625** [comment]
hey, can you please elaborate on how you plan this?

**Comment by Nikita110602** [comment]
@sunny0625 Yes, I have a idea to add few information and data to event and program section, By adding a new page in which there is information about the Open Source Events happening around the globe month-wise and programs listed month-wise.

**Comment by sunny0625** [comment]
What say @AnirudhPanda ?

**Comment by AnirudhPanda** [comment]
Still couldn’t get it how it is different from the existing one
